config/locales/candidate_interface/interruption.yml

```
@@ -0,0 +1,27 @@
+en:
+  review_interruption:
+    title: Give your application the best chance of success
+    statement_count: Your personal statement is %{word_count}.
+    success_percentage: 90% of successful candidates write about %{word_count} words or more for their personal statement.
+    increase_chances: Increase your chances of getting an offer by adding more information to your personal statement.
+    edit_statement: Would you like to edit your personal statement before submitting your application?
+    edit_statement_button_text: Edit your personal statement
+    continue_without_editing: Continue without editing
+  review_enic_interruption:
+    title: You have not included a UK ENIC reference number
+    waiting_maybe:
+      enic_link_text: apply for a statement of comparability from UK ENIC (opens in new tab)
+      planned_link_html: You have said you had, or planned. to %{link} but you have not added the reference number to this application.
```

```suggestion
      planned_link_html: You have said you had, or planned to %{link} but you have not added the reference number to this application.
```

Is there a stray full-stop here?